app/renderer/components/main/main.js

```diff
@@ -758,6 +791,30 @@ const styles = StyleSheet.create({
 
   tabPagesWrap_allowDragging: {
     WebkitAppRegion: 'drag'
+  },
+
+  deprecationBanner: {
+    backgroundColor: 'white',
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    padding: '8px 20px'
+  },
+
+  deprecationBanner_Message: {
+    color: globalStyles.color.commonTextColor,
+    fontSize: '15px'
```

Comment:
this needs to have `cursor: default` and `user-select: none` added (it currently shows a text selection cursor and is selectable via double click)